webknossos/webknossos/utils.py

```diff
@@ -213,34 +212,21 @@ def warn_deprecated(deprecated_item: str, alternative_item: str) -> None:
     )
 
 
-def make_upath(maybe_path: Union[str, PathLike, Path]) -> UPath:
-    return maybe_path if isinstance(maybe_path, UPath) else UPath(maybe_path)
-
-
 def is_fs_path(path: Path) -> bool:
```

Comment:
We could get rid of this as well, but I actually think `is_fs_path` is more expressive than `not isinstance(path, UPath)`.